### ISSUE TITLE
add a flag to mark automatically answered questions

### ISSUE THREAD

**Issue by Gebbi8** [FAIR-CA-indicators/fair-ca-indicators-backend#8]

**Comment by Fancien** [comment]
We need to discuss this, at the moment there is a flag for disabled assessments (either because of the pre-session form or because of dependencies), and I added a flag for automated assessments.

But we need to decide when this `automated` flag should be set to True, and if we need other ones

**Comment by Gebbi8** [comment]
Well, everything that can not be changed by direct user interaction should be flagged as automated.

- questions that are answered based on the preleminary assessment
- questions which answers are derived from their dependencies

**Comment by Gebbi8** [comment]
Is the automated flag is currently always set to false?

**Comment by Fancien** [comment]
> Is the automated flag is currently always set to false?

Yes, I added the attribute but it is not modified yet in the code.

**Comment by Fancien** [comment]
> Well, everything that can not be changed by direct user interaction should be flagged as automated.
> 
>     * questions that are answered based on the preleminary assessment
> 
>     * questions which answers are derived from their dependencies

I just added the change in the `develop` branch. When the task is created, I set the `automated` attribute to `True` if the default status is not `queued`. This covers both cases.
